Source/inv.h

```diff
@@ -4,6 +4,7 @@
 
 extern BOOL invflag;
 extern BOOL drawsbarflag;
+extern const InvXY InvRect[73];

```

Comment:
This doesn't appear to be used outside of SourceX/miniwin/misc_msg.cpp

Reply:
It's defined in `inv.cpp` and used in `plrctrls.cpp` for inventory navigation and in `misc_msg.cpp` for focusing on the first inventory cell.

Reply:
Why not define it in plrctrls.h?

Reply:
It's from the original game, most of the handling is in `inv.cpp`.